src/main.js

```diff
@@ -218,6 +218,9 @@ function configureCommandlineSwitchesSync(cliArgs) {
 		}
 	});
 
+	// Enable Font Access API (eg. for terminal ligatures)
+	app.commandLine.appendSwitch('enable-features', 'FontAccess');
```

Comment:
`FontAccess` feature is enabled by default in the runtime from Chromium 104 https://chromium-review.googlesource.com/c/chromium/src/+/3653487, can you add a TODO saying to remove this once we update to Electron 20, thanks!